app_server/test/services/UserService.js

```diff
@@ -85,6 +85,24 @@ lab.experiment('UserService tests', function () {
     });
   });
 
+  lab.test('getUserById valid arguments', function(done) {
+    UserService.createNewUser(bob).then(function (result) {
+      return UserService.getUserById(result.userId);
+    }).then(function(user) {
+      Code.expect(user.username).to.equal(bob.username);
+      Code.expect(user.password).to.equal(bob.password);
+      done();
+    });
+  });
+
+  lab.test('getUserById invalid arguments', function(done) {
```

Comment:
Yeah, that's what I thought too, but I'm not quite sure how to even implement that check.
Maybe leave some sort of comment for future reference regardless of whichever method is used, I guess.